### ISSUE TITLE
Additional empty class when generating Enum in SpecificRecord format

### ISSUE THREAD

**Issue by francescopellegrini** [julianpeeters/sbt-avrohugger#49]
The following `.avdl` schema:

```
@namespace("namespace.avro")
protocol EnumProtocol {
    enum MyEnum {
        VALUE1,
        VALUE2
    }
}
```

using the following settings:

```
sourceGenerators in Compile += (avroScalaGenerateSpecific in Compile).taskValue
```

is generated correctly, but I also get a `EnumProtocol.scala` file in the same package with the following body:

```
/** MACHINE-GENERATED FROM AVRO SCHEMA. DO NOT EDIT DIRECTLY */
package namespace.avro

import scala.annotation.switch
```

which causes compiler to produce some warnings:

```
[warn] Found names but no class, trait or object is defined in the compilation unit.
[warn] The incremental compiler cannot record the dependency information in such case.
[warn] Some errors like unused import referring to a non-existent class might not be reported.
```

Is this behavior known?

**Comment by julianpeeters** [comment]
Hi @francescopellegrini Hmm that is unexpected. Thanks for the schema, I'll try to see what went wrong this weekend. I'm surprised the tests didn't catch that. It's possible that I forgot to `clean` before testing the release you're using, I might try version 1.1.0 if you're looking to troubleshoot.

**Comment by julianpeeters** [comment]
Thanks again for reporting this @francescopellegrini, avrohugger was indeed generating an extra scala file for the protocol when there was only a java enum defined within. This is fixed now in avrohugger version 1.0.0-RC7 and sbt-avrohugger 2.0.0-RC7
